include/arch/posix/barrier.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ARCH_POSIX_BARRIER_H_
+#define ZEPHYR_INCLUDE_ARCH_POSIX_BARRIER_H_
+
+#ifndef _ASMLANGUAGE
+
+/* Not implemented yet */
```

Comment:
would you expect anything else in this arch than what the general barrier.h defines?

Reply:
not for posix. The comment is indeed a bit misleading :)